cmd/cli/kubectl-kyverno/apply/apply_command_test.go

```diff
@@ -110,21 +111,85 @@ func Test_Apply(t *testing.T) {
 				},
 			},
 		},
+		{
+			config: ApplyCommandConfig{
+				PolicyPaths:   []string{"../../../../test/best_practices/disallow_latest_tag.yaml"},
+				ResourcePaths: []string{"-"},
+				PolicyReport:  true,
+				AuditWarn:     true,
+			},
+			stdinFile: "../../../../test/resources/pod_with_latest_tag.yaml",
+			expectedPolicyReports: []preport.PolicyReport{
+				{
+					Summary: preport.PolicyReportSummary{
+						Pass:  1,
+						Fail:  0,
+						Skip:  0,
+						Error: 0,
+						Warn:  1,
+					},
+				},
+			},
+		},
+		{
+			config: ApplyCommandConfig{
+				PolicyPaths:   []string{"https://github.com/kyverno/policies/openshift/team-validate-ns-name/"},
+				ResourcePaths: []string{"../../../../test/openshift/team-validate-ns-name.yaml"},
```

Comment:
github policy mode is not any more testinga agains a local cluster but a yaml test file

Reply:
What do you mean by "not testing against a local cluster"?

Why do we need to specify ResourcePaths here? Shouldn't it fetch all from remote?

Reply:
it seem the comment is not pinnend to the correct line.

The previous version hat cluster= true set and was checking the resources agains a local cluster, which is not available during uniot tests. see: https://github.com/kyverno/kyverno/blob/9d3b176defe4b7faeb6b4f664e40eef9151cb8a4/cmd/cli/kubectl-kyverno/apply/apply_command_test.go#L80

Reply:
The Implementation fetches the policies from remote, but not the resources to test.
As it was previousely set to Cluster = true, ther reources checked came from the local cluster.

No the check is not any more against the cluster, hence it needs soome resource files to test.

Reply:
Makes sense now!